src/frontend/src/lib/components/ui/LogoWithCorner.svelte

```diff
@@ -0,0 +1,24 @@
+<script lang="ts">
+	import Logo from '$lib/components/ui/Logo.svelte';
+	import type { Token } from '$lib/types/token';
+
+	export let token: Token;
+	export let size: string;
```

Comment:
If this component becomes meant for `TokenLogo` we can be less generic here and expose for example a property `export let size: 'small' | 'big' = 'big';`.

Reply:
You mean the `Logo` component right? I had a look throughout the code and it is used elsewhere too. However, the list of all sizes in which it is used is:

- 64px
- 52px
- 20px

So, I can modify it to have  'small' | 'medium' | 'big', what do you think? In any case, it will be in another PR

Reply:
Done, please check last commit. It is mostly used as `'small'` , so maybe we can default to that?